Code/CMakeLists.txt

```diff
@@ -316,6 +316,31 @@ if(SV_USE_OPENCASCADE)
 	#set(SV_INSTALL_EXTERNALS ON)
 	find_package(FREETYPE)
 endif()
+
+#-----------------------------------------------------------------------------
+# GDCM
+if(SV_USE_GDCM)
+	simvascular_external(GDCM DOWNLOADABLE)
+	mark_as_superbuild(GDCM_DIR)
+	mark_as_superbuild(GDCM_SOURCE_DIR)
+	set(USE_GDCM ON)
+	set(GLOBAL_DEFINES "${GLOBAL_DEFINES} -DSV_USE_GDCM")
+	#set(SV_EXTERNAL_SHARED_LIBS ${SV_EXTERNAL_SHARED_LIBS} GDCM)
+	#set(SV_INSTALL_EXTERNALS ON)
+endif()
+
+#-----------------------------------------------------------------------------
+# FREETYPE
+if(SV_USE_FREETYPE)
+	simvascular_external(FREETYPE DOWNLOADABLE)

```

Comment:
What you mentioned is the issue. You are not adding the external project for superbuild if not using system or a specified FREETYPE. We can get around this temporarily if instead you use SYSTEM_DEFAULT instead of DOWNLOADABLE.